### DISCUSSION TITLE
One web api for getting power status + energy status

### DISCUSSION THREAD

**Discussion by silentbil** [arendst/Tasmota#10260] [Q&A] [answered]
Hi,
im currently need to do 2 get requests
 1.  /cm?cmnd=Power    - to know if its on/off
 2. /cm?cmnd=status%208  - to get all the power consumption information

is there a way to get both in the same api request?

**Comment by sfromis** [comment, marked as answer]
Commands via http is not really an API, merely a very secondary feature not intended as "the way" of using devices. MQTT is the preferred option, where you do not even need to query such info, but just subscribe to the messages. No effort has been (or likely will be) made to make a flexible REST-like API out of the barebones http command support.

**Comment by ascillato** [comment]
Use command backlog
